redis_tasks/registries.py

```diff
@@ -114,12 +114,12 @@ def get_dead_ids(self):
             self.key, '-inf', oldest_valid))
 
 
-worker_registry = WorkerRegistry()
+worker_registry = LazyObject(lambda: WorkerRegistry())
```

Comment:
we dont need a lambda here, right?
```suggestion
worker_registry = LazyObject(WorkerRegistry)
```

Reply:
right, we could do like so. Thought that that way would be analogous to the rest, but maybe that is not better